test.html

```
@@ -0,0 +1,16 @@
+<html>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)